velox/runner/LocalRunner.cpp

```diff
@@ -289,4 +281,23 @@ std::vector<exec::TaskStats> LocalRunner::stats() const {
   return result;
 }
 
+std::vector<SplitSource::SplitAndGroup> TestingSplitSource::getSplits(
+    uint64_t targetBytes) {
+  std::vector<SplitSource::SplitAndGroup> result;
+  if (splitIdx_ >= splits_.size()) {
+    return {{nullptr, 0}};
+  }
+  result.push_back(SplitAndGroup{std::move(splits_[splitIdx_++]), 0});
```

Comment:
since we only return one split and let just return it as vector with single item directly